workers/pipeline/create_pipeline.go

```diff
@@ -124,7 +125,11 @@ func CreatePipeline(p *gaia.CreatePipeline) {
 	}
 
 	nsUUID := uuid.NewV4()
-	p.Pipeline.TriggerToken = uuid.NewV5(nsUUID, "pipelineTriggerToken").String()
```

Comment:
@michelvocks I just realised that this was hardcoded namespace. Although the previous UUID was pseudo-random, hardcoding the namespace here is like giving it a seed of some sort. It was probably fine because of the random UUID but... you never know. :) Unless I would have wanted to create a hierarchy, I just feel I will sleep better knowing that it's not. :D

Reply:
@Skarlso Thanks for pointing that out. I'm currently working on the secret generation for the worker and I would like to use the same functionality you use here. Maybe we should refactor that to an external place? Maybe `security`?

Reply:
@michelvocks Hmm, okay, sure. That's a good suggestion. I'll do it.